publication-commons/src/test/resources/features/AnonymousAccessRights.feature

```diff
@@ -0,0 +1,62 @@
+Feature: Anonymous access rights
+
+  Background:
+    Given that there is a database "PUBLICATIONS" with publications
+    And that DynamoDBPublicationService exists
+    And the DynamoDBPublicationService has a READ method
+    And the READ method takes as parameter a non empty username
```

Comment:
Do we just care about username or should we provide some context object with username, roles etc.?